src/backend/aspen/api/views/phylo_runs.py

```diff
@@ -196,73 +188,37 @@ async def kick_off_phylo_run(
     return PhyloRunResponse.from_orm(workflow)
 
 
-async def _get_accessible_phylo_runs(db, user, run_id=None, editable=False):
-    # get phylo_runs viewable or editable by a user, optionally filtered by id
-    cansee_owner_group_ids: Set[int] = {
-        cansee.owner_group_id
-        for cansee in user.group.can_see
-        if cansee.data_type == DataType.TREES
-    }
-    query = sa.select(PhyloRun).options(
-        joinedload(PhyloRun.outputs.of_type(PhyloTree)),
+async def get_serializable_runs(
```

Comment:
minor refactor of the previous `_get_accessible_phylo_runs` interface. This is just a shortcut method to get all phyloruns from the db that a user has a given `permission` level for.